src/master/browsers.json

```diff
@@ -47,6 +47,14 @@
         "os_version": "10.13",
         "remote": true
     },
+    "safari-11.1-macos-10.13": {
```

Comment:
Is this really right? Isn't it Safari 12?

Reply:
You're right. This technically works because the key doesn't influence any behavior. I'm going with `safari-stable-macos` to mirror the other entries and to limit the overhead of maintaining the correct version number since [I'm not very good at that](https://github.com/web-platform-tests/results-collection/commit/60a0bc91968ee4e3503508c7ea8f5d2aaa3db111#diff-e66b0e85335625519fa3847b00e226fd).